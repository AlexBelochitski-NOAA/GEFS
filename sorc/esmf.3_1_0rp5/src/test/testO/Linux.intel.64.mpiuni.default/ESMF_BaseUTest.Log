20190610 112231.230682 INFO     PET0 Running with ESMF Version 3.1.0rp5
20190610 112231.230945 INFO     PET0  Beginning Test, file ESMF_BaseUTest.F90, line          71
20190610 112231.230962 INFO     PET0  NUMBER_OF_PROCESSORS           1
20190610 112231.230997 INFO     PET0  PASS  ESMF_BaseCreate, ESMF_BaseUTest.F90, line          80
20190610 112231.231007 INFO     PET0  PASS  ESMF_Destroy, ESMF_BaseUTest.F90, line          89
20190610 112231.231021 INFO     PET0  Number of failed tests:           0
20190610 112231.231027 INFO     PET0  Ending Test, file ESMF_BaseUTest.F90, line         232
