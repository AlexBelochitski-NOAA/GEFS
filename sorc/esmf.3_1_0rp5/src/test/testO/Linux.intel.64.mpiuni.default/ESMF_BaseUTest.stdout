  Beginning Test, file ESMF_BaseUTest.F90, line          71
  NUMBER_OF_PROCESSORS           1
  PASS  ESMF_BaseCreate, ESMF_BaseUTest.F90, line          80
  PASS  ESMF_Destroy, ESMF_BaseUTest.F90, line          89
  Number of failed tests:           0
  Ending Test, file ESMF_BaseUTest.F90, line         232
